docs/book/src/getting-started/testdata/project/PROJECT

```diff
@@ -3,6 +3,7 @@
 # and allow the plugins properly work.
 # More info: https://book.kubebuilder.io/reference/project-config.html
 domain: example.com
+kubebuilderVersion: (devel)
```

Comment:
If we use goreleaser and simulate a release and then run kubebuilder init with the binary, are we able to ensure that it will have the version instead of devel?